src/dashboard/src/components/ingest/views.py

```diff
@@ -494,10 +564,34 @@ def transfer_backlog(request):
         if models.SIPArrange.objects.filter(
             original_path__endswith=path['relative_path']).exists():
             not_draggable = True
-        _es_results_to_directory_tree(path['relative_path'], return_list, not_draggable=not_draggable)
+        if ui == 'legacy':
+            _es_results_to_directory_tree(path['relative_path'], return_list, not_draggable=not_draggable)
+            response = return_list
+        else:
+            _es_results_to_appraisal_tab_format(path, directory_map, return_list, not_draggable=not_draggable)
+            response = {
+                "formats": [],  # TODO populate this
+                "transfers": return_list
+            }
 
     # retun JSON response
-    return helpers.json_response(return_list)
+    return helpers.json_response(response)
+
+
+@decorators.elasticsearch_required()
+def transfer_backlog(request):
+    """
+    AJAX endpoint to query for and return transfer backlog items.
+    """
+    return transfer_request(request, 'legacy')
+
+
+@decorators.elasticsearch_required()
+def transfer_appraisal(request):
+    """
+    AJAX endpoint to query for and return transfer backlog items in the appraisal tab format.
+    """
+    return transfer_request(request, 'appraisal')
```

Comment:
Probably cleaner to use urls.py to pass the extra parameter and not have the wrapping functions. https://docs.djangoproject.com/en/1.7/topics/http/urls/#passing-extra-options-to-view-functions